sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -155,6 +155,28 @@ static void hda_dma_reset(struct snd_sof_dev *sdev, void *dma_data)
 	snd_hdac_ext_stream_reset(hext_stream);
 }
 
+static int hda_dai_config(struct snd_sof_dev *sdev, struct snd_soc_dapm_widget *w,
+			  int channel, unsigned int flags)
```

Comment:
what does 'channel' represent?

Reply:
this change is removed now. I've reused the existing tplg IPC op instead